3-probability/06-normal-approx/extra-practice.qmd

```diff
@@ -0,0 +1,91 @@
+---
+title: "Continuous Distributions and Normal Approximations"
+subtitle: "EP"
+format: stat20handout-pdf
+---
+
+\begin{center}
+\textbf{This problem set will not be graded. Its materials will be tested on Quiz 2. We recommend that you work on it as study material.}
+\end{center}
+
+@. The function below is a valid probability density function. *Hint:* The area of a circle is $\pi r^2$ where $r$ is the radius of the circle.
+
+```{r}
+#| echo: false
+#| fig-height: 5
+#| fig-width: 5
+#| fig-align: left
+knitr::include_graphics("images/circle.png")
+```
+
+$\textcolor{white}{space space space}$ $\bigcirc$ True $\textcolor{white}{space space space}$ $\bigcirc$ False
+
+
+We are going to study what I would call a "pessimistic random walk". Say you are in a one-dimensional space, and you can do one of three things, randomly, at any given moment:
```

Comment:
@shomurali and I discussed how in a future semester, we might try to take out random walks just because its a next-level of complexity that's a bit out of scope for Stat 20. Totally fine as is though.

Reply:
Sounds good!